src/lib/v35.js

```diff
@@ -49,9 +52,9 @@ module.exports = function(name, version, hashfunc) {
   } catch (err) {
   }
 
-  // Pre-defined namespaces, per Appendix C
-  generateUUID.DNS = '6ba7b810-9dad-11d1-80b4-00c04fd430c8';
-  generateUUID.URL = '6ba7b811-9dad-11d1-80b4-00c04fd430c8';
+  // For CommonJS default export support
+  generateUUID.DNS = DNS;
+  generateUUID.URL = URL;
```

Comment:
Maybe have the v3/v5.js files explicitly `export DNS, URL from `./lib/v35.js` rather than bodging it in here?  (Would that work?)

Reply:
I tried that but it does not seem to work.

I'm using https://www.npmjs.com/package/babel-plugin-add-module-exports to add `module.exports` to the transpiled CommonJS modules so people can continue to do

```js
const uuidv4 = require('uuid/v4');
```

instead of having to do ugly workarounds like

```js
const uuidv4 = require('uuid/v4').default;
```

or

```js
const { default: uuidv4 } = require('uuid/v4');
```

As soon as there is more than `export default …` this `module.exports` approach will no longer work.